src/kubernetes.cc

```diff
@@ -650,17 +651,22 @@ struct Watcher {
 #endif
       return boost::iterator_range<const char*>(iter, end);
     } else if (iter == end) {
+      std::string line(begin, end);
       LOG(ERROR) << "Invalid chunked encoding: '"
                  << std::string(begin, end)
-                 << "'";
-      return boost::iterator_range<const char*>(begin, end);
+                 << "'; exiting";
```

Comment:
Something like `"'; Skipping the current chunk and exiting the thread. Metadata of certain monitored resource might not be up to date.";`
?

Not very sure if the impact is accurate.

Reply:
Take a look at the new message. WDYT?